services/workflows-service/src/alert/dtos/decision-alert.dto.ts

```diff
@@ -1,9 +1,20 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { AlertState } from '@prisma/client';
-import { IsEnum } from 'class-validator';
-import { AlertsIdsByProjectDto } from './assign-alert.dto';
+import { ArrayMinSize, IsArray, IsEnum, IsString } from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class AlertDecisionDto {
+  @ApiProperty({
+    // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
What caused the neccessity of this 2 `eslint-disable`? What's wrong with `Array<String>`?

Reply:
Some of our team members don't have ESLint on save running. ESLint thinks we're trying to use a type where in reality its a runtime use. Once I save its autofixed to `Array<string>` which is invalid as a runtime value.

Reply:
How can we fix it? Will it be fixed if we add ESLint as a pre-commit hook (if it's not already set)?

Reply:
Yes. This can be a real blow to committing though so it should be discussed. I would like to avoid giving people a reason to use `--no-verify`.